modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavaPlayFrameworkCodegen.java

```diff
@@ -0,0 +1,250 @@
+package io.swagger.codegen.languages;
+
+import io.swagger.codegen.*;
+import io.swagger.codegen.languages.features.BeanValidationFeatures;
+
+import java.io.File;
+import java.util.List;
+import java.util.Map;
+
+public class JavaPlayFrameworkCodegen extends AbstractJavaCodegen implements BeanValidationFeatures {
+
+    public static final String TITLE = "title";
+    public static final String CONFIG_PACKAGE = "configPackage";
+    public static final String BASE_PACKAGE = "basePackage";
+    public static final String CONTROLLER_ONLY = "controllerOnly";
+    public static final String SINGLE_CONTENT_TYPES = "singleContentTypes";
+    public static final String RESPONSE_WRAPPER = "responseWrapper";
+    public static final String USE_TAGS = "useTags";
+
+    protected String title = "swagger-petstore";
+    protected String configPackage = "io.swagger.configuration";
+    protected String basePackage = "io.swagger";
+    protected boolean controllerOnly = false;
+    protected boolean singleContentTypes = false;
+    protected String responseWrapper = "";
+    protected boolean useTags = false;
+    protected boolean useBeanValidation = true;
+
+    public JavaPlayFrameworkCodegen() {
+        super();
+        outputFolder = "generated-code/javaPlayFramework";
+        apiTestTemplateFiles.clear();
+        embeddedTemplateDir = templateDir = "JavaPlayFramework";
+        apiPackage = "controllers";
+        modelPackage = "apimodels";
+        invokerPackage = "io.swagger.api";
+        artifactId = "swagger-java-playframework";
+
+        projectFolder = "";
+        sourceFolder = projectFolder + File.separator + "app";
+        projectTestFolder = projectFolder + File.separator + "test";
+        testFolder = projectTestFolder;
+
+        additionalProperties.put(CONFIG_PACKAGE, configPackage);
+        additionalProperties.put(BASE_PACKAGE, basePackage);
+
+        //TODO: Validate this?
+        // play framework uses the jackson lib, why have a parameter for this?
+        additionalProperties.put("jackson", "true");
```

Comment:
Ok I will remove this one because I'm not using it, it's a copy paste from the "spring" generator